### ISSUE TITLE
Unexpected BatchNorm layer

### ISSUE THREAD

**Issue by ToniButland1998** [microsoft/nni#5232]
**Describe the issue**:
![img](https://user-images.githubusercontent.com/42618317/202596923-059e2389-68a1-44cb-83dd-0ca9e73c8fa1.png)

I used QAT quantized and pass a **dummy_input** into the quantizer. But the net work I got, compared the original no quantized model, has another bn_module batchNorm layer after each Conv2D layer. Therefore, although my quantized pytorch model can work, my exported ONNX model can't generated a reasonable result(the onnx model doesn't this newly added bn_module batch norm structure). This picture is the comparison between original network structure and quantized structure.

My quantize config is  :

Config_list = [{
'quant_tyes': ['input'],
'quant_bits': {'input': 8},
'op_types': ['Conv2d']
}]

I read the source code, they said this is to simulate the batchNorm folding. But why they added another BN, instead of FOLD it?

Thank you for your help.

**Comment by J-shang** [comment]
hello @ToniButland1998 , in fact, these two bn is the same obj, in `QuantizerIdentityWrapper`, it will directly return input in forward:
```
class QuantizerIdentityWrapper(torch.nn.Module):
    def __init__(self, module, module_name):
        """
        Used to wrap modules that should be treated as torch.Identity

        Parameters
        ----------
        module : pytorch module
            the module to be wrapped
        module_name : str
            the name of the module to wrapped, wrapper module shares same name
        """
        super().__init__()
        self.module = module
        self.module_name = module_name

    def forward(self, x):
        return x
```
and all the wrapper is for simulating the quantization effect, if you want to export to ONNX, please using `quantizer.export_model` to get the quantized model weight and the calibration_config, then load the model with the quantized weight and then export to ONNX. i.e.:
```
calibration_config = quantizer.export_model('model_state_save_path')
model = init_model()
model.load_state_dict(torch.load('model_state_save_path'))
```

**Comment by ToniButland1998** [comment]
@ [J-shang](https://github.com/J-shang) Thank you very much for your advice. My question is about '**quantizer.export_model()**'. I think I can only use it once in one script, according to its property. However, during training, we usually need to export it for many times for different epochs. Therefore, I would like to know how can we use '**quantizer.export_model()**' for multiple times? Thank you for your help.

**Comment by J-shang** [comment]
Closed due to no reply for a long time, please reopen if have any questions.
